core/src/main/java/tc/oc/pgm/community/features/VanishManagerImpl.java

```diff
@@ -145,6 +177,10 @@ public void checkMatchPlayer(MatchPlayerAddEvent event) {
     event.getPlayer().setVanished(isVanished(event.getPlayer().getId()));
   }
 
+  private boolean isVanishSubdomain(String address) {
+    return address.startsWith("vanish"); // TODO Maybe allow config value?
```

Comment:
No need for config, I think this is ok.

Reply:
I'd make it `vanish.`, otherwise it may match more than you want, i can only imagine some `vanished.network` having trouble with it